source/fundamentals/builders/aggregates.txt

```diff
@@ -975,3 +975,39 @@ aggregation stage:
 Learn more about this helper from the
 `searchMeta() API documentation <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/Aggregates.html#searchMeta(com.mongodb.client.model.search.SearchCollector)>`__.
 
+Atlas Vector Search
+-------------------
+
+.. important::
+
+   To learn what versions of MongoDB Atlas support this feature, see
+   (TODO: link atlas-vector-search/vector-search-stage/#limitations after those docs are published)
+
+Use the ``vectorSearch()`` method to create a ``$vectorSearch`` (TODO: link to Server manual once updated)
+pipeline stage that specifies a **semantic search**. A semantic search is
+a type of search which locates information that is similar in meaning.
```

Comment:
I think semantic search is purposely kept unspecific because it could be any kind of data. Generally, you should be able to provide it different representations. 

For vector search, a type of semantic search, you convert some data (could be a single term, entire books, images, sound files, etc.) into tokens. The model represents tokens as "embeddings" in a vector data structure. In the code example, the vector is stored in the "queryVector" variable.